spec/features/user/account/update_spec.rb

```diff
@@ -1,25 +1,25 @@
 require "rails_helper"
 
-feature "Update account with valid data" do
+feature "Update Account" do
   let(:user) { create :user, :confirmed }
 
-  before(:each) do
-    sign_in(user.email, "123456")
+  before do
```

Comment:
yep, thanks. Fixed all other consistency issues in https://github.com/fs/rails-base/commit/4d74e519714ee44b667bccf25e950ba4af581b86

Please take another look at this PR.